src/health_misinfo_shared/youtube_api.py

```diff
@@ -104,18 +104,32 @@ def load_texts(folder) -> list[dict]:
     return flat_list
 
 
-def form_chunks(transcript: list[dict]) -> Iterator[str]:
-    """Split/merged a list of sentences into series of overlapping text chunks."""
+def form_chunks(transcript_obj: dict) -> Iterator[dict]:
+    """Split/merged a list of sentences into series of overlapping text chunks.
+    Each chunk is a dict containing the text and the start/end timestamps"""
     current_chunk_text = ""
-    for sentence in transcript:
-        current_chunk_text += sentence["sentence_text"] + " "
-        if len(current_chunk_text) > 5000:
-            yield current_chunk_text
+    current_chunk_start_offset = 0.0
+    current_chunk_end_offset = 0.0
+    for s in transcript_obj:
+        current_chunk_text += s["sentence_text"] + " "
+        if len(current_chunk_text) > 1500:
```

Comment:
Just noting this change to chunk size. Is this deliberate? If the LLM is happy with this, then that’s great – I suspect these smaller chunks will work much better for the offsets.